hydra-node/src/Hydra/ContestationPeriod.hs

```diff
@@ -20,13 +20,11 @@ instance Arbitrary ContestationPeriod where
   arbitrary = do
     UnsafeContestationPeriod . fromInteger
       <$> oneof
-        [ choose (1, confirmedHorizon)
+        [ elements [0, 1]
+        , choose (1, confirmedHorizon)
         , pure confirmedHorizon
         , choose (confirmedHorizon, oneDay)
-        , pure oneDay
-        , pure oneWeek
-        , pure oneMonth
-        , pure oneYear
+        , elements [oneDay, oneWeek, oneMonth, oneYear]
```

Comment:
FWIW this is not strictly the same as the old code! It will change the frequency of these values. I think it's acceptable, but I wonder why you did change it.

Reply:
It was part of previous _experiments_. I am reverting it to the old instance we had.